database/migrations/2021_09_05_201643_create_missed_approach_notifications_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateMissedApproachNotificationsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('missed_approach_notifications', function (Blueprint $table) {
+            $table->id();
+            $table->string('callsign')->index()->comment('The callsign the notification relates to');
```

Comment:
Is there a case for a callsign, created_at combined index here? or maybe callsign,expires_at?